lib/api/contracts/model_contract.rb

```diff
@@ -51,6 +51,27 @@ def self.attribute(*attributes, &block)
       validate :readonly_attributes_unchanged
       validate :run_attribute_validations
 
+      # Performs validation of both, the contract and the underlying model and returns
+      # the merged validation errors for both of them.
+      def validate_contract_and_model
+        self.validate
+        model.valid?
+
+        errors = self.errors.dup
```

Comment:
I am pretty sure that the assignment reaches the local variable called errors.

But since you are asking the question in the first place, I see that there is room for confusion which I did not intend, so I'll rename it ^^

P.S.: I probably did not notice because of my code highlighting, which differenticates between local variables and instance methods.